vnext/Microsoft.ReactNative.Managed.CodeGen.UnitTests/AnalysisTestBase.cs

```diff
@@ -91,7 +91,7 @@ protected async Task<(CodeAnalyzer, INamedTypeSymbol)> AnalyzeFileAsync(string c
             // ways one can run inttests, this seemed to be the most reasonable out of a lot of poor options.
             var win10SdkFolder = @"C:\Program Files (x86)\Windows Kits\10";
 #if win10SdkVersion10_0_18362_0
```

Comment:
I don't follow some of the assertions in the comment but where is this #define coming from? can't find it defined anywhere and now it would be mismatched with the value below

Reply:
Neither did I.

May be an outdated and possibly no longer used macro.

I'll ping the core team about this before merging.